snipssonos/services/service.py

```diff
@@ -0,0 +1,6 @@
+from snipssonos.helpers.snips_config_parser import read_configuration_file
+
+
+class Service(object):
+    CONFIG = "config.ini"
+    CONFIGURATION = read_configuration_file(CONFIG)
```

Comment:
I think we could add an improvement to lower code duplication. Here you could define a dummy HOST and PORT and a `get_port_from_config` and `get_host_from_config` methods. The methods would be called with the service name to pick up info from the config. 
You would have this in this class: 
~~~python
def get_port_from_config(service_name):
      self.PORT = self.CONFIGURATION['global'][service_name] if (
          self.CONFIGURATION['global'][service_name]) else self.PORT
~~~
and for instance in /music/search_service.py you could have: 
~~~python
self.PORT = self.get_port_from_config('music_service_port')
~~~
WDYT?

Reply:
That's a good idea !

Reply:
Actually, there's a huge huge side effect in my implementation. We are basically reading from the file system every time we create an instance of any service .... 
This is bad, I will find something better.